client/src/components/Settings.jsx

```diff
@@ -2,6 +2,7 @@ import React, { useState } from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
+import TCLink from '@material-ui/core/Link';
```

Comment:
Let's just call think `Link`

Reply:
Sure. I did try this originally, but got an error saying "Identifier "Link" has already been declared" since we're also using Link from react-router. Is there a work-around?

Reply:
You could update the "Update Information" button to not use the `Link` component and instead use `useHistory`:

```jsx
const history = useHistory();

// ...

<Button className={classes.updateButton} onClick={() => history.push('/onboard')}>
  Update Information
</Button>
```

Reply:
That worked! Changes are in.